refs/types.proto

```diff
@@ -77,6 +77,18 @@ message OwnerID {
   bytes value = 1 [json_name = "value"];
 }
 
+// NeoFS subnetwork identifier.
+//
+// `SubnetID` is a 4-byte long.
```

Comment:
Suffice it to say that it corresponds to the standard NeoFS binary format which is Protocol Buffers with direct orders of fields (I'm not sure it is actually 4-byte long because of prefixes).

Reply:
Doc shouldn't really describe the format (we are talking about `.proto` file after all), but having explicit id size is something I would like to know from the doc. Moved this to the field comment.